rascal-lsp/src/main/rascal/util/LanguageServer.rsc

```diff
@@ -202,6 +218,7 @@ data LanguageService
     | definer(Definer definer)
     | referrer(Referrer reference)
     | implementer(Implementer implementer)
+    | codeActionContributor(CodeActionContributor actions)
```

Comment:
Somehow `codeActionContributor` seems like name that is different than the other names in this list, can we find something that matches the rest?

Reply:
I'll think about that. It's true this one is the odd one out.

Reply:
inlayHinter has the same problem. 

If we follow `lenses` then:
* `inlayHinter` should be `inlays`
* `codeActionContributor` should be `actions` or `codeActions`

Any preference? I think `actions` is nice and short. That it's about code is "nogal wiedes".